app/design/adminhtml/default/default/template/ebizmarts/mailchimp/sales/order/view/monkey.phtml

```diff
@@ -27,7 +27,7 @@
                             ); ?>
                             </strong>
                         <?php else: ?>
-                            <strong><?php echo $this->quoteEscape(
+                            <strong><?php echo $this->getMailChimpHelper()->mcEscapeQuote(
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.